### DISCUSSION TITLE
Dual color lightning

### DISCUSSION THREAD

**Discussion by Xenomorphys** [seerge/g-helper#2106] [Ideas]
Hello, 
First of all, a big thank you for your software, much lighter and simpler than Armory...
I just went through it and I'm missing one option (small but nice): the possibility of putting two colors and a gradient in the LED parameters.
Many thanks if you can add that, and even without that, thank you! 
![IMG_20240221_220152](https://github.com/seerge/g-helper/assets/87600477/a2596ce9-2138-403a-9498-db537c286cd2)

**Comment by seerge** [comment]
@Xenomorphys hello,

Do you mean custom backlight for that back-window? then i was already answering this in a few issues, unfortunately i don't know how it's controlled.

So someone with actual device and armory needs to manually debug and intercept usb-commands that armory sends when sets color there

**Reply by Xenomorphys** [reply]
Not necessarily ONLY the rear window, but I know that on Aura, we can make a gradient and it's quite pretty.

**Reply by seerge** [reply]
@Xenomorphys you have single-zone keyboard, so what kind of gradient do you mean there ? If it's just changing color, then on hardware level keyboard supports only rainbow and breathing (2 colors) :) All other "modes" would be emulated on software side by Armoury by constantly (few times a second) spamming a new color to keyboard. And I'm not a big fan of that at it would  consume resources / battery ...

**Reply by Xenomorphys** [reply]
I understand, I didn't know about the keyboard.
So, that for the rear window.
Afterwards, I don't know how long it would take to put this in place.
I don't want to take time off from work for such a trivial thing.
In any case, I thank you for your software!
And thank you for your help 😁
